src/LockFreeIndexSetBuilders.cpp

```diff
@@ -198,24 +193,19 @@ void buildLockFreeBlockIndexset(
 /*
  ******************************************************************************
  *
- * Build Lock-free "color" index set. The domain-set is colored based on
- * connectivity to the range-set.  All elements in each segment are
- * independent, and no two segments can be executed in parallel.
- *
- * Note: Method assumes IndexSet ptr refers to an empty index set.
+ * Generate a lock-free "color" index set containing range and list segments.
  *
  ******************************************************************************
  */
 void buildLockFreeColorIndexset(
-    RAJA::TypedIndexSet<RAJA::RangeSegment,
-                        RAJA::ListSegment,
-                        RAJA::RangeStrideSegment>& iset,
+    RAJA::TypedIndexSet<RAJA::RangeSegment, RAJA::ListSegment>& iset,
+    camp::resources::Resource& work_res,
     Index_type const* domainToRange,
     int numEntity,
     int numRangePerDomain,
     int numEntityRange,
-    Index_type* elemPermutation,
-    Index_type* ielemPermutation)
+    Index_type* elemPermutation = 0l,
+    Index_type* ielemPermutation = 0l)
```

Comment:
should these defaults be nullptr here and in the header?

Reply:
Probably. This is really old code and I don't recall what it does. I don't think anyone use the lock-free indexset stuff in RAJA anymore because the execution stuff has been removed -- there were special forall methods for that about 5 years ago..  I was going to ping users and see if anyone uses this stuff and see if folks think it can/should be removed.